box-content-sdk/src/main/java/com/box/androidsdk/content/models/BoxSession.java

```diff
@@ -242,6 +242,15 @@ public BoxSharedLinkSession getSharedLinkSession(String sharedLinkUri) {
     }
 
     /**
+     * This function gives you the cache location associated with this session. It is
+     * preferred to use this method when setting up the location of your cache as it ensures
+     * that all data will be cleared upon logout.
+     */
+    public String getCacheLocation(){
+        return getApplicationContext().getFilesDir().getPath()+"/"+getUserId();
+    }
```

Comment:
Perhaps it might be better returning the actual directory. 
return new File(getApplicationContext().getFilesDir(), getUserId());

and for consistency maybe call it getCacheDir